src/nghttp.cc

```diff
@@ -687,7 +687,9 @@ void HttpClient::disconnect() {
   state = ClientState::IDLE;
 
   for (auto req = std::begin(reqvec); req != std::end(reqvec); ++req) {
-    (*req)->continue_timer->stop();
+    if ((*req)->continue_timer) {
```

Comment:
We have not had test suite for nghttp yet.  We have nghttpx integration test suite written in Go, so we can do the same sort of thing for nghttp.